package.json

```diff
@@ -9,19 +9,14 @@
     "format:base": "prettier \"./**/*.{ts,js}\"",
     "format:fix": "npm run format:base -- --write",
     "format:check": "npm run format:base -- --check",
-    "test": "nyc mocha --timeout=30000",
-    "test:unit": "mocha --exclude test/git.spec.js",
-    "coverage": "nyc report --reporter=lcov",
+    "test": "jest --verbose --testTimeout 30000 --silent",
```

Comment:
It might be better to put `testTimeout` (and maybe the others) in the `jest` config, to avoid having two places to look.

Reply:
yes makes sense